awxkit/awxkit/cli/docs/README.md

```diff
@@ -28,6 +28,8 @@ Building the Documentation
 --------------------------
 To build the docs, spin up a real AWX server, `pip install sphinx sphinxcontrib-autoprogram`, and run:
 
-    ~ TOWER_HOST=https://awx.example.org TOWER_USERNAME=example TOWER_PASSWORD=secret make clean html
-    ~ cd build/html/ && python -m http.server
+    cd $AWX_FOLDER # folder where the ansible/awx repository has been cloned to
```

Comment:
I'd remove this line - I assume if you're reading this README, you're aware the docs live here 😄